helm/values.yaml

```diff
@@ -3,7 +3,7 @@ fullnameOverride: ""
 
 image:
   repository: mongo
-  tag: 4.2.7
+  tag: 4.4.0
```

Comment:
Are we planning to upgrade client libraries as well? Generally recommended.

Reply:
Yes, will update next but that's not a blocker for this AFAIK.